docs/spicedb-dedicated/dedicated-isolation.md

```diff
@@ -0,0 +1,11 @@
+# SpiceDB Dedicated Isolation
+
+SpiceDB Dedicated isolates your workloads from other customers. Cloud accounts, compute resources, databases, and networking are all dedicated to you.
+
+Additionally, SpiceDB allows you to deploy multiple isolated Permissions Systems into a single SpiceDB Dedicated environment. Each Permissions Systems has it’s own schema and set of relationships. Also, these Permissions Systems impose memory and CPU limits so one Permissions System can’t crowd out another Permissions System.
```

Comment:
Additionally, SpiceDB *Dedicated allows

Reply:
Typo in "Each Permissions System*s" should be "Permissions (or Permission pending Jimmy) System"